fr/appendices/fixture-upgrade.rst

```diff
@@ -0,0 +1,155 @@
+Mise à Niveau des Fixtures
+##########################
+
+À partir de la version 4.3.0, le schéma de fixture et les responsabilités de
+gestion des données sont scindés.
+Maintenir le schéma dans les classes de fixtures et les migrations rajoutait de
+la complexité et un coût de maintenance pour les applications. Dans 4.3.0, de
+nouvelles APIs ont été ajoutées pour permettre de réutiliser plus facilement les
+migrations existantes ou des outils de gestion de schéma avec des tests.
+
+Pour mettre à niveau vers le nouveau système de fixtures, vous devrez faire
+quelques mises à jour:
+
+#. Tout d'abord, enlevez le bloc ``<listeners>`` de votre fichier
+   ``phpunit.xml``.
+#. Ajoutez ceci à votre fichier ``phpunit.xml``::
+
+        <extensions>
+            <extension class="\Cake\TestSuite\FixtureSchemaExtension" />
+        </extensions>
+
+   Cela enlève la gestion de schéma du gestionnaire de fixture de test. À la
```

Comment:
The indentation here is off. This line and the following one should not be indented.

Reply:
Note that the indentation is exactly the same in english. Please confirm or dismiss.

Reply:
This looks ok.

Reply:
My mistake, I was using github's syntax highlighting which is a bit off.